mu-plugins/plugin-loader.php

```diff
@@ -15,6 +15,7 @@
  */
 function create_wordpress_project_core_plugins(): array {
 	return [
+		// INSTALLED_PLUGINS_PLACEHOLDER.
```

Comment:
I think we should DRY up this file and use `wp-plugin-loader` here instead of letting projects fragment themselves with a custom plugin loader

Reply:
@srtfisher I agree... I thought this file was using `wp-plugin-loader` for some reason.